src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -4656,73 +4644,7 @@ void M()
-                // (6,33): error CS1525: Invalid expression term 'stackalloc'
-                //         int*[] p = new int*[] { stackalloc int[1] };
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "stackalloc").WithArguments("stackalloc").WithLocation(6, 33));
-        }
-
-        [Fact]
-        public void CS1575ERR_BadStackAllocExpr3()
-        {
-            // Diff errors
-            var test = @"
-unsafe public class Test
-{
-    void M()
-    {
-        const int* p = stackalloc int[1];
-    }
-}
-";
-            CreateStandardCompilation(test, options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary, allowUnsafe: true)).VerifyDiagnostics(
-                // (6,15): error CS0283: The type 'int*' cannot be declared const
-                //         const int* p = stackalloc int[1];
-                Diagnostic(ErrorCode.ERR_BadConstType, "int*").WithArguments("int*").WithLocation(6, 15),
-                // (6,24): error CS1525: Invalid expression term 'stackalloc'
-                //         const int* p = stackalloc int[1];
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "stackalloc").WithArguments("stackalloc").WithLocation(6, 24));
-        }
-        
-        public void CS1674ERR_StackAllocInUsing1()
-        {
-            // Diff errors
-            var test = @"
-public class Test
-{
-    unsafe public static void Main()
-    {
-        using (var v = stackalloc int[1])
-        {
-        }
-    }
-}
-";
-            CreateStandardCompilation(test, options: TestOptions.ReleaseDll.WithAllowUnsafe(true)).VerifyDiagnostics(
-                // (6,16): error CS1674: 'int*': type used in a using statement must be implicitly convertible to 'System.IDisposable'
-                //         using (var v = stackalloc int[1])
-                Diagnostic(ErrorCode.ERR_NoConvToIDisp, "var v = stackalloc int[1]").WithArguments("int*").WithLocation(6, 16));
-        }
-
-        [Fact]
-        public void CS0029ERR_StackAllocInUsing2()
-        {
-            // Diff errors
-            var test = @"
-public class Test
-{
-    unsafe public static void Main()
-    {
-        using (System.IDisposable v = stackalloc int[1])
-        {
-        }
-    }
-}
-";
-            CreateStandardCompilation(test, options: TestOptions.ReleaseDll.WithAllowUnsafe(true)).VerifyDiagnostics(
-                // (6,39): error CS0029: Cannot implicitly convert type 'int*' to 'System.IDisposable'
-                //         using (System.IDisposable v = stackalloc int[1])
-                Diagnostic(ErrorCode.ERR_NoImplicitConv, "stackalloc int[1]").WithArguments("int*", "System.IDisposable").WithLocation(6, 39));
+            CreateStandardCompilation(test, options: new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary, allowUnsafe: true)).GetParseDiagnostics().Verify();
         }
```

Comment:
>  } [](start = 6, length = 3)

Possible I missed this but where do we have tests that tie stackalloc changes to our language version flag?